src/modules/commander/HealthAndArmingChecks/checks/distanceSensorChecks.cpp

```diff
@@ -48,7 +48,7 @@ void DistanceSensorChecks::checkAndReport(const Context &context, Report &report
 
 		if (exists) {
 			distance_sensor_s dist_sens;
-			valid = _distance_sensor_sub[instance].copy(&dist_sens) && hrt_elapsed_time(&dist_sens.timestamp) < 1_s;
+			valid = _distance_sensor_sub[instance].copy(&dist_sens) && hrt_elapsed_time(&dist_sens.timestamp_sample) < 1_s;
```

Comment:
Probably overthinking it, but in this case I think `timestamp` is appropriate because we're just checking if the publication is recent. Maybe somewhat contrived, but you could have a fairly delayed reading coming from somewhere. 

```suggestion
			valid = _distance_sensor_sub[instance].copy(&dist_sens) && hrt_elapsed_time(&dist_sens.timestamp) < 1_s;
```